guides/common/modules/con_content-view-environments-overview.adoc

```diff
@@ -0,0 +1,21 @@
+[id="content-view-environments-overview"]
+= Content view environments overview
+
+The *Allow multiple content views* setting controls whether hosts and activation keys can be assigned to multiple content view environments.
+By default, this feature is disabled, limiting assignments to a single content view environment.
+Enable this setting to assign multiple content view environments to hosts and activation keys.
+For more information on the *Allow multiple content views* setting, see xref:multiple-activation-keys-and-content-view-environments[].
```

Comment:
It seems odd to link to this here. I'd delete it, since we have a correct link on the next line.